packages/breadcrumbs/src/elements/Breadcrumb.spec.tsx

```diff
@@ -72,9 +73,12 @@ describe('Breadcrumb', () => {
 
       items.forEach((item, i) => {
         if (i === lastItemIndex) {
-          expect(item.parentElement).toHaveClass('is-current');
+          expect(item.parentElement).toHaveStyleRule(
```

Comment:
I don't think this has to be changed, just want to surface another testing strategy I had to use with CSS-in-JS.

We have a babel transform which removes all `data-test-*` attributes in production builds. If you didn't want to assert a color style here you could also use a `data-test-current="true|false"` with `toHaveAttribute()` if it makes more sense.

Reply:
Nice, thanks for surfacing another testing strategy!

The test above this one, (`applies current aria values correctly`) also verifies that the last item is current by asserting that it has `aria-current=page`. The `toHaveStyleRule` is used to verify the color style because this test description, (`receives current styling if last`), involves the current styling.